tests/utilities/test_xla_device_utils.py

```diff
@@ -16,40 +16,32 @@
 import pytest
 
 import pytorch_lightning.utilities.xla_device_utils as xla_utils
-from pytorch_lightning.utilities import XLA_AVAILABLE
+from pytorch_lightning.utilities import XLA_AVAILABLE, TPU_AVAILABLE
 from tests.base.develop_utils import pl_multi_process_test
 
 if XLA_AVAILABLE:
     import torch_xla.core.xla_model as xm
 
 
-@pytest.mark.skipif(XLA_AVAILABLE, reason="test requires torch_xla to be absent")
+@pytest.mark.skipif(TPU_AVAILABLE, reason="test requires torch_xla to be absent")
 def test_tpu_device_absence():
     """Check tpu_device_exists returns None when torch_xla is not available"""
     assert xla_utils.XLADeviceUtils.tpu_device_exists() is None
```

Comment:
this is tricky testing itself and for the testing set "if I am true test that I am true :D"

Reply:
ah.. there seems to be a mismatch here from my local. I basically made this into
```python
@pytest.mark.skipif(not xla_utils.XLADeviceUtils.xla_available(), reason="test requires torch_xla to be installed")
```

Not sure how I pushed this code.